app/views/layouts/admin/_header.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="wrapper">
+  <!-- Preloader -->
+  <div class="preloader flex-column justify-content-center align-items-center">
+    <img class="animation__shake" src="dist/img/AdminLTELogo.png" alt="AdminLTELogo" height="60" width="60">
+  </div>
+  <!-- Navbar -->
+  <nav class="main-header navbar navbar-expand navbar-white navbar-light">
+    <!-- Left navbar links -->
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <a class="nav-link" data-widget="pushmenu" href="#" role="button"><i class="fas fa-bars"></i></a>
+      </li>
+      <li class="nav-item d-none d-sm-inline-block">
+        <a href="#" class="nav-link"> <%= t ".home" %> </a>
+      </li>
+      <li class="nav-item d-none d-sm-inline-block">
+        <a href="#" class="nav-link"> <%= t ".contact" %> </a>
+      </li>
+      <li class="nav-item d-none d-sm-inline-block">
+        <%= link_to t("layouts.header.en"), locale: "en" %>
+      </li>
+      <li class="nav-item d-none d-sm-inline-block">
+        <%= link_to t("layouts.header.vi"), locale: "vi" %>
+      </li>
+    </ul>
+    <!-- Right navbar links -->
+    <ul class="navbar-nav ml-auto">
+      <!-- Navbar Search -->
+      <li class="nav-item">
+        <a class="nav-link" data-widget="navbar-search" href="#" role="button">
+          <i class="fas fa-search"></i>
+        </a>
+        <div class="navbar-search-block">
+          <form class="form-inline">
+            <div class="input-group input-group-sm">
+              <input class="form-control form-control-navbar" type="search" placeholder="Search" aria-label="Search">
+              <div class="input-group-append">
+                <button class="btn btn-navbar" type="submit">
+                  <i class="fas fa-search"></i>
+                </button>
+                <button class="btn btn-navbar" type="button" data-widget="navbar-search">
+                  <i class="fas fa-times"></i>
+                </button>
+              </div>
+            </div>
+          </form>
+        </div>
+      </li>
+    </ul>
+  </nav>
```

Comment:
e k tự check dc cái này à ?
nhìn nó bị thiếu thẻ đóng đùng đùng
thẻ mở ở file nào thì thẻ đóng ở file đó

Reply:
Hic layout này chuổi thật chị ạ, nó có <div class="wrapper"> để bọc toàn bộ content. Em update lại vào application rồi ạ